gui/utils/compat.py

```diff
@@ -1,262 +0,0 @@
-# Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
```

Comment:
Mmmm, I'm going to leave this one for now.  Taking a quick look at it, `OrderedDict` is fairly intertwined, and I don't want to make this PR any more complex.

I did nuke the import of the 2.6 `urllib` though. :)